azure/v31.1.1/README.md

```diff
@@ -0,0 +1,361 @@
+
+### Apps
+
+- azure-cloud-controller-manager from v1.30.14-gs1 to v1.31.8-gs1
+- azure-cloud-node-manager from v1.30.14-gs1 to v1.31.8-gs1
+- azuredisk-csi-driver from v1.30.12-gs1 to v1.31.11-gs1
+- azurefile-csi-driver from v1.30.10-gs1 to v1.31.7-gs1
+- capi-node-labeler from v1.0.2 to v1.1.2
+- cert-exporter from v2.9.5 to v2.9.8
+- cert-manager from v3.9.0 to v3.9.1
+- cilium from v0.31.5 to v1.2.2
+- coredns from v1.24.0 to v1.26.0
+- etcd-defrag from v1.0.2 to v1.0.6
+- etcd-k8s-res-count-exporter from v1.10.3 to v1.10.6
+- k8s-audit-metrics from v0.10.2 to v0.10.5
+- k8s-dns-node-cache from v2.8.1 to v2.9.0
+- net-exporter from v1.22.0 to v1.23.0
+- node-exporter from v1.20.2 to v1.20.4
+- observability-bundle from v1.11.0 to v2.0.0
+- observability-policies from v0.0.1 to v0.0.2
+- security-bundle from v1.10.1 to v1.12.0
+- teleport-kube-agent from v0.10.4 to v0.10.6
+- vertical-pod-autoscaler from v5.4.0 to v5.5.1
+- vertical-pod-autoscaler-crd from v3.2.0 to v3.3.1
+
+
+### azure-cloud-controller-manager [v1.30.14-gs1...v1.31.8-gs1](https://github.com/giantswarm/azure-cloud-controller-manager-app/compare/v1.30.14-gs1...v1.31.8-gs1)
+
+#### Changed
+
+- Chart: Update to upstream v1.31.8. ([#112](https://github.com/giantswarm/azure-cloud-controller-manager-app/pull/112))
+
+### azure-cloud-node-manager [v1.30.14-gs1...v1.31.8-gs1](https://github.com/giantswarm/azure-cloud-node-manager-app/compare/v1.30.14-gs1...v1.31.8-gs1)
+
+#### Changed
+
+- Chart: Update to upstream v1.31.8. ([#105](https://github.com/giantswarm/azure-cloud-node-manager-app/pull/105))
+- Update PolicyExceptions apiVersion to v2.
+
+### azuredisk-csi-driver [v1.30.12-gs1...v1.31.11-gs1](https://github.com/giantswarm/azuredisk-csi-driver-app/compare/v1.30.12-gs1...v1.31.11-gs1)
+
+#### Changed
+
+- Chart: Update to upstream v1.31.11.
+
+### azurefile-csi-driver [v1.30.10-gs1...v1.31.7-gs1](https://github.com/giantswarm/azurefile-csi-driver-app/compare/v1.30.10-gs1...v1.31.7-gs1)
+
+#### Changed
+
+- Chart: Update to upstream v1.31.7.
+
+### capi-node-labeler [v1.0.2...v1.1.2](https://github.com/giantswarm/capi-node-labeler-app/compare/v1.0.2...v1.1.2)
+
+#### Changed
+
+- Go: Update dependencies.
+- Go: Update dependencies.
+- Improve Control Plane node detection.
+- Taint Control Plane nodes if not already tainted.
+- Go: Update dependencies.
+
+### cert-exporter [v2.9.5...v2.9.8](https://github.com/giantswarm/cert-exporter/compare/v2.9.5...v2.9.8)
+
+#### Changed
+
+- Go: Update dependencies.
+- Go: Update dependencies.
+- Fix linting issues.
+- Go: Update dependencies.
+
+### cert-manager [v3.9.0...v3.9.1](https://github.com/giantswarm/cert-manager-app/compare/v3.9.0...v3.9.1)
+
+#### Added
+
+- Added Vertical Pod Autoscaler support for `controller` pods.
+- Added renovate configutarion
+
+#### Removed
+
+- Removed dependabot configuration
+
+### cilium [v0.31.5...v1.2.2](https://github.com/giantswarm/cilium-app/compare/v0.31.5...v1.2.2)
```

Comment:
I think here's it's the same the changelog for Cilium also includes more than what's in between the two versions

Reply:
The PR https://github.com/giantswarm/devctl/pull/1431 might help in case you don't wanna go through it manually.